modules/apps/frontend-js/frontend-js-react-web/package.json

```diff
@@ -14,7 +15,7 @@
 		"react-dnd-test-backend": "11.1.1",
 		"react-dnd-test-utils": "11.1.1"
 	},
-	"main": "js/index.es.js",
+	"main": "js/index.js",
```

Comment:
Is this really happening? 😱  ❤️

Reply:
Well in this case it is because the file got renamed from `index.es.js` to `index.ts` (wouldn't have made sense to call it `index.es.ts`). And that in turn means that the built file is now called `index.js`.

Technically a "breaking" change if anybody is referring to this by name, but nobody should be and I can't find any instances of usage... (and if they are, well, what the heck are they doing?)